packages/commonwealth/client/scripts/utils/trustLevel.ts

```diff
@@ -0,0 +1,103 @@
+import { COMMUNITY_TIER, USER_TIER } from '@hicommonwealth/schemas';
+
+interface TrustLevel {
+  level: number;
+  icon: string;
+  name: string;
+  description: string;
+}
+
+export interface UserTrustLevel extends TrustLevel {}
+export interface CommunityTrustLevel extends TrustLevel {}
+
+const USER_TRUST_LEVELS: Record<number, UserTrustLevel> = {
+  0: {
```

Comment:
What do these tiers correspond with? On the platform side, tiers are different:

```
const tierLimitsPerHour = [
  // tier 0 (new unverified addresses with no sso, balance, etc - bad aura)
  { create: 0, upvote: 0, ai: { images: 0, text: 0 } },
  // tier 1 (new verified address)
  { create: 1, upvote: 5, ai: { images: 2, text: 5 } },
  // tier 2 (1 week old and verified)
  { create: 2, upvote: 10, ai: { images: 4, text: 10 } },
  // tier 3 (email verified/sso, or wallet with balance (TODO: cache balances))
  { create: 5, upvote: 25, ai: { images: 10, text: 50 } },
];
```

User tier 0 should not be mapped on the client. It will not happen. A user cannot be signed in with an unverified address that is impossible.

Reply:
We should have a single `USER_TIERS` object (in `libs/shared`)  that both the client and the platform use so we can avoid confusion.

Reply:
my config is directly taken from [here ](https://www.figma.com/design/jveQc0C5XtW0Mq8v1sBSnD/Verification?node-id=1-55&t=Eh6B5NZcVi9zJrSi-1). 

![image](https://github.com/user-attachments/assets/16e69d86-0c75-4912-ad11-dba1f6e39d96)

Agree we should have common config across the board. Open to discussions. I also use only icon from config now but wanted it to be flexible and ready to extend. 

@Rotorsoft wdyt?

Reply:
yep, we should "share" all common definitions

Reply:
Also not a fan of these emojis lol - there is no flow from one to the next. Having a baby emoji??? Icons for this should be easily understandable and to me these are not.

Think about icons for ranks in competitive games -> these have clear continuity, a user can tell which user is higher level simply by looking at the icon _without having to look up the definition of these icons_.

Reply:
used shared config